runtime/oti/shcflags.h

```diff
@@ -50,7 +50,7 @@
 #define J9SHR_DATA_TYPE_CACHELET  7
 #define J9SHR_DATA_TYPE_ZIPCACHE 8
 #define J9SHR_DATA_TYPE_JITHINT 9
-#define J9SHR_DATA_TYPE_UNUSED1 10
+#define J9SHR_DATA_TYPE_GC 10
```

Comment:
Perhaps in the future there will be other types of hints. This should be more specific, like J9SHR_DATA_TYPE_GC_HEAPHINT_2. I added the 2 for the two heap sizes.

Reply:
Fixed